lambdas/tests/unit/services/test_bulk_upload_report_service.py

```diff
@@ -439,13 +439,21 @@ def test_generate_summary_report_with_two_ods_reports(
 def test_generate_success_report_writes_csv(
     bulk_upload_report_service, mock_get_times_for_scan, mocker
 ):
+    mock_file_name = (
+        f"daily_statistical_report_bulk_upload_success_{MOCK_TIMESTAMP}.csv"
+    )
     bulk_upload_report_service.write_and_upload_additional_reports = mocker.MagicMock()
 
     test_ods_reports = bulk_upload_report_service.generate_ods_reports(
         MOCK_REPORT_ITEMS_ALL,
     )
 
     bulk_upload_report_service.generate_success_report(test_ods_reports)
+    expected = readfile("expected_success_report.csv")
```

Comment:
Isn't this the logic that's being tested below in the test on line 588? Is there another file being created here? We don't want to test the logic that's done as part of `write_and_upload_additional_reports()`, we just want to test it's being called, I just pulled that out into a separate test last week.

Reply:
Discussed this on a call - Steph & Abid's approach avoids a much more cumbersome & verbose refactor. The existing logic did not check what was actually in the csv that was created.